src/test/java/com/hubspot/jinjava/el/ext/RegularDivTest.java

```diff
@@ -0,0 +1,27 @@
+package com.hubspot.jinjava.el.ext;
+
+import static org.junit.Assert.assertEquals;
+
+import com.hubspot.jinjava.Jinjava;
+import java.util.HashMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RegularDivTest {
+  private Jinjava jinja;
+
+  @Before
+  public void setUp() {
+    jinja = new Jinjava();
+  }
+
+  @Test
+  public void okOnLongDivisionWithNonZeroDivisor() {
+    assertEquals(jinja.render("{% set x = 10 / 2%}{{x}}", new HashMap<>()), "5.0");
```

Comment:
Does the normal division operator return division of longs with a decimal point?

Reply:
(Cut-and-pasted from my reply to "Did you have a question ..." comment above). When I tried coding RegularDivOperator to return a long on division of long arguments I got a regression failure on the ExpressionResolverTest.itResolvesMathVals(), which was expecting a result of "5.0" when dividing 15L by 3L. Switching to having RegularDivOperator always return a double fixed the regression and yielded a clean build.